composer.json

```diff
@@ -20,7 +20,9 @@
         "cakephp/cakephp": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.*"
+        "phpunit/phpunit": "4.1.*",
+        "squizlabs/php_codesniffer": "2.*",
```

Comment:
This is not needed. `cakephp/cakephp-condesniffer` already has it as a dependency.